docs/csharp/language-reference/compiler-messages/cs1614.md

```diff
@@ -8,7 +8,7 @@ helpviewer_keywords:
 ms.assetid: 787eef18-74a6-49d9-9f34-f0881aca8dff
 ---
 # Compiler Error CS1614
-'name' is ambiguous; between 'attribute1' and 'attribute2'. use either '@attribute' or 'attributeAttribute'  
+Description: 'name' is ambiguous between 'name' and 'nameAttribute'; use either '@name' or 'nameAttribute'.
```

Comment:
@johnnyagerard please remove "Description: " at the start of the paragraph as it's not the part of the error message. Typically, the first paragraph of a compiler error article is the citation of the corresponding error message.

Reply:
@pkulikov Alright.